pipeline/tests/pil.rs

```diff
@@ -90,9 +94,7 @@ fn permutation_with_selector() {
 #[test]
 fn fibonacci() {
     let f = "pil/fibonacci.pil";
-    verify_pil(f, Default::default());
-    test_halo2(f, Default::default());
-    gen_estark_proof(f, Default::default());
```

Comment:
pil and estark would re-run witgen from scratch even though the parameters are the same. Now the pipeline is re-used for pil and estark.